src/winetricks

```diff
@@ -2741,38 +2741,51 @@ w_call()
     w_try w_do_call "$@"
 }
 
+w_backup_reg_file()
+{
+    W_reg_file=$1
+
+    w_get_sha256sum "$W_reg_file"
+
+    w_try cp "$W_reg_file" "$W_TMP_EARLY"/_reg_"$(echo "$_W_gotsha256sum" | cut -c1-8)"_$$.reg
```

Comment:
Sorry I missed this before, but the quoting could be simplified, remove the quote after early and _reg_.

After that, ready for merge!

Reply:
Yes, well spotted. I fixed it.